File: service/model/spu.go
package model

type Spu struct {
	SpuID     int64  `json:"spuID"`
	SpuCode   string `json:"spuCode"`
	SpuDesc   string `json:"spuDesc"`
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
}

File: fontend/MyAntDesignApp2/Models/Spu.cs
using System.Collections.Generic;
using System.ComponentModel;

namespace WmsApp.Models
{
    public class Spu : CommonDate
    {
        public long SpuID { get; set; }
        [DisplayName("编码")]
        public string SpuCode { get; set; }
        public string SpuDesc { get; set; }
        public List<Sku> SkuList { get; set; }
    }

}
